include/alpaka/dev/cpu/SysInfo.hpp

```diff
@@ -156,21 +176,17 @@ namespace alpaka
                 std::uint64_t size(0);
                 std::size_t sizeLen{sizeof(size)};
                 if(sysctl(mib, 2, &size, &sizeLen, nullptr, 0) < 0)
-                {
                     throw std::logic_error("getTotalGlobalMemSizeBytes failed calling sysctl!");
-                }
                 return static_cast<std::size_t>(size);
 
 #    elif defined(_SC_AIX_REALMEM) // AIX.
                 return static_cast<std::size_t>(sysconf(_SC_AIX_REALMEM)) * static_cast<std::size_t>(1024);
 
-#    elif defined(_SC_PHYS_PAGES) && defined(_SC_PAGESIZE) // Linux, FreeBSD, OpenBSD, Solaris.
-                return static_cast<std::size_t>(sysconf(_SC_PHYS_PAGES))
-                    * static_cast<std::size_t>(sysconf(_SC_PAGESIZE));
+#    elif defined(_SC_PHYS_PAGES) // Linux, FreeBSD, OpenBSD, Solaris.
+                return static_cast<std::size_t>(sysconf(_SC_PHYS_PAGES)) * getPageSize();
 
-#    elif defined(_SC_PHYS_PAGES) && defined(_SC_PAGE_SIZE) // Legacy.
-                return static_cast<std::size_t>(sysconf(_SC_PHYS_PAGES))
-                    * static_cast<std::size_t>(sysconf(_SC_PAGE_SIZE));
+#    elif defined(_SC_PHYS_PAGES) // Legacy.
+                return static_cast<std::size_t>(sysconf(_SC_PHYS_PAGES)) * getPageSize();
```

Comment:
Now we have two `#elif defined(_SC_PHYS_PAGES)`. I think we can merge those.

Reply:
@psychocoderHPC You missed this review comment. We have two branches doing the same thing here.

Reply:
you are right :-(